python/cudf/cudf/core/dataframe.py

```diff
@@ -5891,6 +5891,41 @@ def _from_columns(cls, cols, index=None, columns=None):
 
         return cls(data=data, index=index,)
 
+    @copy_docstring(Frame._interpolate)
+    def interpolate(
+        self,
+        method="linear",
+        axis=0,
+        limit=None,
+        inplace=False,
+        limit_direction=None,
+        limit_area=None,
+        downcast=None,
+        **kwargs,
+    ):
+
+        if (
+            method in {"index", "values"}
+            and not self.index.is_monotonic_increasing
+        ):
+            warnings.warn("Unsorted Index...")
```

Comment:
Should we do this? What should we put here?

Reply:
As usual, `pandas` seems OK with some pretty nonsensical cases:
```
In [83]: pd.Series([2, None, 4, None, 2], index=[1, 2, 3, 2, 1]).interpolate('values')
Out[83]:
1    2.0
2    3.0
3    4.0
2    3.0
1    2.0
dtype: float64
```